modules/styling-transform/lib/utils/createStyleObjectNode.ts

```diff
@@ -64,10 +65,9 @@ export function serializeStyles(
 ) {
   const {getFileName, styles, cache, names, extractedNames} = context;
   const fileName = getFileName(node.getSourceFile().fileName);
-  const hash = generateUniqueId(); //getHash(node, context);
-  const serialized = {...serializedStylesEmotion([input]), name: hash} as ReturnType<
-    typeof serializedStylesEmotion
-  >;
+  const serialized = serializedStylesEmotion([input]);
+  const hash = getHash(node, context);
+  serialized.name = hash;
```

Comment:
Instead of using a non-deterministic auto-incremented ID, we use a deterministic hash based on the AST node (start and end character index)